frontend/src/components/explorer/mapViewer/DataOverlay.vue

```diff
@@ -148,6 +146,11 @@ export default {
   computed: {
     ...mapState({
       model: state => state.models.model,
+      showing2D: state => state.maps.showing2D,
+      dataOverlayPanelVisible: state => state.maps.dataOverlayPanelVisible,
+    }),
+    ...mapGetters({
+      queryParams: 'maps/queryParams',
```

Comment:
I don't see where this is being used in this file. Do we need it?

Reply:
I think it should be used to read from the state the value of `dataOverlayPanelVisible`.

Reply:
Sorry, I don't know why 4 lines got marked here. I meant to only mark line 153.

Reply:
Gotcha, the getter is removed through [this commit](https://github.com/MetabolicAtlas/MetabolicAtlas/pull/512/commits/0fee62e60fa16ba8b9278268058ec15fcc7d9b02).